packages/material-ui/src/locale/index.js

```diff
@@ -106,6 +106,27 @@ export const frFR = {
   },
 };
 
+export const idID = {
+  props: {
+    MuiTablePagination: {
+      backIconButtonText: 'Halaman sebelumnya',
+      labelRowsPerPage: 'Baris per halaman:',
+      labelDisplayedRows: ({ from, to, count }) => `${from}-${to === -1 ? count : to} of ${count}`,
+      nextIconButtonText: 'Halaman selanjutnya',
+    },
+    MuiRating: {
+      getLabelText: value => `${value} Star${value !== 1 ? 's' : ''}`,
```

Comment:
Any idea how we could translate this one?

Reply:
getLabelText: value => `${value} Bintang`,